test/JDBC/input/babel_test_int4_numeric_oper_before_16_3-vu-verify.mix

```diff
@@ -1,3 +1,4 @@
+-- sla 50000
```

Comment:
whats the earlier execution-time?

Reply:
previously it was around 22145ms, currently it is around 37142ms.